assets/models/system/deformable_detr_twostage_refine_r50_16x2_50e_coco/description.md

```diff
@@ -4,7 +4,7 @@ DETR has been recently proposed to eliminate the need for many hand-designed com
 
 > The above abstract is from MMDetection website. Review the <a href="https://github.com/open-mmlab/mmdetection/tree/v2.28.2/configs/deformable_detr" target="_blank">original-model-card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
```

Comment:
Needs alignment

Reply:
Or since this is getting deprecated, we don't need to make changes?

Reply:
Yes Ayush. We are not updating the deprecated model cards.